doc/cookbook/source/examples/gaussian_processes/gaussian_process_regression.rst

```diff
@@ -0,0 +1,59 @@
+===========================
```

Comment:
Really like the cookbook page!
I think we should shorten it slightly. See below